source/reflect.h

```diff
@@ -3661,9 +3661,9 @@ parse_context_branch_reset_state::parse_context_branch_reset_state(){}
 #line 1993 "reflect.h2"
     [[nodiscard]] auto parse_context::grab_n(cpp2::impl::in<int> n, cpp2::impl::out<std::string> r) & -> bool
     {
-        if (cpp2::impl::cmp_less_eq(pos + cpp2::impl::as_<size_t>(n),regex.size())) {
-            r.construct(regex.substr(pos, cpp2::impl::as_<size_t>(n)));
-            pos += (cpp2::impl::as_<size_t>(n)) - 1;
+        if (cpp2::impl::cmp_less_eq(pos + cpp2::unsafe_narrow<size_t>(n),regex.size())) {
```

Comment:
these days you can just write `unsafe_narrow` (without the `cpp2::`), I think.

Reply:
I don't know, I will try. This code should be generated by cppfront... I will check how it should be generated.

Reply:
cppfront generates it in that way:
```cpp
        if (cpp2::impl::cmp_less_eq(pos + cpp2::unsafe_narrow<size_t>(n),regex.size())) {
            r.construct(regex.substr(pos, cpp2::unsafe_narrow<size_t>(n)));
            pos += (cpp2::unsafe_narrow<size_t>(n)) - 1;
            return true; 
```

Reply:
My bad, I meant to put the review inside the `.h2` file.